src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -2047,18 +2047,10 @@ def validate_upload_blob(namespace):
 
 
 def add_upload_progress_callback(cmd, namespace):
-    def _update_progress(response):
-        if response.http_response.status_code not in [200, 201]:
-            return
-
-        message = getattr(_update_progress, 'message', 'Alive')
-        reuse = getattr(_update_progress, 'reuse', False)
-        current = response.context['upload_stream_current']
-        total = response.context['data_stream_total']
-
+    def _update_progress(current, total):
```

Comment:
This seems duplicated with https://github.com/Azure/azure-cli/blob/76faac6276b056dd5c3f49d7b9196824ef190907/src/azure-cli/azure/cli/command_modules/storage/_validators.py#L1015-L1030

Can you reuse the existing one?

Reply:
Done.